include/camp/config.in.hpp

```diff
@@ -5,6 +5,8 @@
 #cmakedefine CAMP_ENABLE_CUDA
 #cmakedefine CAMP_ENABLE_HIP
 #cmakedefine CAMP_ENABLE_SYCL
+#cmakedefine CAMP_WIN_STATIC_BUILD
+#cmakedefine CAMP_DLL_EXPORTS
```

Comment:
Is this going to be confusing with the 'CAMP_DLL_EXPORT' macro constant defined below?

Reply:
The DLL_EXPORTS one shouldn't be exported here, we want that so that we export while compiling ourself and import while compiling others.

Reply:
Wait, should this cmakedefine be removed then?

Reply:
Not sure. I will defer to you folks.

Reply:
I think so, the first one we need, the second may actually cause problems downstream.

Reply:
```suggestion
```